docs/man5/Makefile.am

```diff
@@ -3,7 +3,8 @@ MAN5_FILES  = \
 
 A2X_ARGS = \
 	-d manpage \
-	-f manpage
+	-f manpage \
+	-L
```

Comment:
probably better "by asking xmllint to validate a dummy xml file against the docbook DTD" than to hardcode OS-specific assumptions that might turn out wrong at the glimpse of an eye.